lib/dfdutils/dfd.h

```diff
@@ -35,7 +35,8 @@ enum VkSuffix {
     s_SINT,    /*!< Signed integer format. */
     s_SFLOAT,  /*!< Signed float format. */
     s_UFLOAT,  /*!< Unsigned float format. */
-    s_SRGB     /*!< sRGB normalized format. */
+    s_SRGB,     /*!< sRGB normalized format. */
```

Comment:
Trivially, removed the space replaced by the comma.
```suggestion
    s_SRGB,    /*!< sRGB normalized format. */
```